packages/atomic/src/components.d.ts

```diff
@@ -75,25 +75,26 @@ export namespace Components {
         "searchStatus": SearchStatus;
     }
     /**
-     * @beta - This component is part of the automatic facets feature.
-     * Automatic facets are currently in beta testing and should be available soon.
      * The `atomic-automatic-facet-generator` is a component that renders the facets from
-     * the automatic facets feature. It dynamically creates multiple atomic-automatic-facet
-     * components based on the desiredCount prop. Automatic facets are generated by the index
+     * the automatic facets feature.
+     * Unlike traditional facets that need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
```

Comment:
If my suspicion was correct about `search query` -> `query`, be sure to apply throughout ✅